app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java

```diff
@@ -421,8 +421,12 @@ private void play(long position, @Nullable Integer transcodedSubtitle) {
                 BaseItemDto item = getCurrentlyPlayingItem();
 
                 // make sure item isn't missing
-                if (item.getLocationType() == LocationType.Virtual && mFragment != null) {
-                    if (hasNextItem()) {
+                if (item == null || item.getLocationType() == LocationType.Virtual) {
+                    if (mFragment == null) {
+                        Timber.d("item is null!");
+                        if (TvApp.getApplication() != null)
+                            Utils.showToast(TvApp.getApplication(), TvApp.getApplication().getString(R.string.msg_cannot_play));
```

Comment:
Don't use TvApp.getApplication....

Reply:
If mFragment is null is there another way to get context?

If not I could just remove the toast

Reply:
If mFragment is null that means that the playbackcontroller is already dead and shouldn't be called anyway.

Reply:
I made a new PR with TvApp.getApplication removed